### PR TITLE
Features

### PR DESCRIPTION
**Pull request by daudi13**
## In this project I:

- Applied the CSS box model
- Applied Flexbox to place elements in the pages.
- Applied media queries to make the site adaptable to different screen sizes
- Applied JavaScript to manipulate DOM elements
- Used JavaScript events
- Used Objects to access data
- Communicated technical concepts to other technical people

Here is a short video description of my project - [video](https://www.loom.com/share/d2ecfb2080c54d2490a64b64d99a6237)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,60 +1,63 @@
...
+- Pure CSS
+- Vanilla JavaScript
+- Github
+- LightHouse
+- Webhint
+- Stylelint
+- EsLint
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
```
Comment (elmar8287) [inline README.md]: - [x] Please, consider fixing the link to your Demo version. As you can there is no possibility to check, if the project deployed via GH pages or not. 💯

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @daudi13 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- Your video is very nice! 👍 
- Styling perfect 👍 

## Required Changes ♻️

- [ ]  Check the comments under the review.
- [ ] Please, add the link of the video into a description of PR. You have added it in a comment.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (daudi13) [reply README.md]: I have updated my README.md file with the link description and I have also Updated my PR description with a link to my video. Let me know if there is something else. Thank you

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @daudi13 👋🏻, 

Your project design looks amazing ❤️! Great job adding the project live link. Keep coding, keep soaring high 🚀 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (elmar8287) [reply README.md]: Nice job done. 👍

**Pull request merged**